### PR TITLE
Korean GP names update

### PR DESCRIPTION
**Pull request by mechu1205**
Deleted a couple of people with somewhat little historical impact.
Added some important historic figures.
Category change for some names.
Fixed spelling for some names.

### CODE REVIEW

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -1148,116 +1148,154 @@ def setup():
...
 		iRenaissance,
-		"Yi Hwang", # 16th
-		"Yi Ji-ham", # 16th
+		"Hwang Hui", # 15th
+		"Ryu Seong-Ryong", # 16th
+		"Yi Won-Ik", # 16th
+		"Che Je-Gong", # 18th
 		iIndustrial,
 		"Heungseon Daewongun", # 19th
-		"Kim Ok-gyun", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:1235]: Wikipedia transliterates Korean names as capital letters only at the beginning of the name, not after the hyphen. Let's stick to that.

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -1148,116 +1148,154 @@ def setup():
...
 		"Kim Gu", # 20th
-		"Kim Dae-jung", # 20th
+		"Kim Dae-Jung", # 20th
+		"Kim Young-Sam", # 20th
 	],
 	iGreatGeneral : [
-		"Gim Yu-sin", # 7th
+		"Gwanggaeto", # 4th
+		"Kim Yu-Sin", # 7th
+		"GyeBaek", # 7th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:1271]: Gyebaek

**Review by dguenms — CHANGES_REQUESTED** [review]

File: Assets/Python/GreatPeople.py (hunk 1 above)
Comment (mechu1205) [reply Assets/Python/GreatPeople.py:1235]: OK

File: Assets/Python/GreatPeople.py (hunk 2 above)
Comment (mechu1205) [reply Assets/Python/GreatPeople.py:1271]: OK

**Pull request closed without merging**